napari/_vispy/_tests/test_vispy_shapes_layer.py

```diff
@@ -6,6 +6,7 @@
 
 def test_remove_selected_with_derived_text():
     """See https://github.com/napari/napari/issues/3504"""
+    np.random.seed(0)
```

Comment:
As I was modifying these tests and know that random shapes can cause flaky tests, I seeded these to make them reproducible.